test/v6/GenerateEIP712Payload.ts

```diff
@@ -0,0 +1,215 @@
+import { Wallet, providers } from "ethers"
+import { SigningHelper } from "@src/v6/signing"
+import { SignatureType, EIP712Signer } from "@src/signing"
+import { AllowFill, GenericSwapData, LimitOrder, RFQOffer, RFQTx } from "@src/v6/types"
+
+describe("Generate V6 EIP712 signature examples", async () => {
```

Comment:
These are more like scripts to generate data, how about turning them into hardhat script, under `v6` directory?

Reply:
Good idea, move to `scripts` now!